### PR TITLE
Gutenboading: remove container max-width

### LINKED ISSUE #40566: Gutenboarding: Page container shouldn't have a max-width
Currently the page container [seems](https://d.pr/i/fzbFmw) to have a `max-width` of `1100px`.

The way we've designed it is to have the page be fluid and use 100% of the width with `88px` padding on each side.

This change goes for all the pages except for the intent screen where we use additional padding (total `120px`). See the design file for more specific measurements.

### PR DESCRIPTION
**Pull request by razvanpapadopol**
#### Changes proposed in this Pull Request

- set margins for desktop (88px) and mobile (20px) as specified in design
- set margins for acquire intent screen as specified (120px)
- left align acquire intent content
- fix acquire intent positioning issues

Screenshots on a 4k screen: [acquire intent](https://cloudup.com/igCTfkhgOzU), [design picker](https://cloudup.com/iLQE-kporYd), [template preview](https://cloudup.com/iWc0Crm5x1i)

#### Testing instructions

* Go to /gutenboarding and checkout the flow
* On larger screens (including 1920px and above), the content should cover the full width of the screen

Fixes #40566

### CODE REVIEW

File: client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-	position: fixed;
 	tab-size: 4;
-	top: 0;
 	transition: color 0.5s ease-in-out 1s, background-color 0.5s ease-in-out 1s;
-	width: 100%;
-}
 
-.acquire-intent__questions {
-	max-width: 1200px;
+	@media ( min-width: 600px ) {
```
Comment (alshakero) [inline client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss]: Maybe this also needs to move to `variables` as the `mobile-breaking-point`.

Comment (razvanpapadopol) [reply client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss]: Good catch! Actually we already have [breakpoints](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries) in Calypso styles so I'm switching to that and we're going to use 660px instead 600px.

Comment (simison) [reply client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss]: we can use breakpoints mixins from [`@wordpres/base-styles`](https://github.com/WordPress/gutenberg/tree/e16260ff98ee4c3163332d7659b50c4fc7a78320/packages/base-styles) package, which we import to Gutenboarding:

https://github.com/Automattic/wp-calypso/blob/2161f3032d5ec6f7d63ef64384958a410774d5d7/client/assets/stylesheets/gutenberg-base-styles.scss#L4-L5

https://github.com/Automattic/wp-calypso/blob/2161f3032d5ec6f7d63ef64384958a410774d5d7/client/landing/gutenboarding/style.scss#L4

Comment (razvanpapadopol) [reply client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss]: Great! 600px it is then 🎯

Comment (simison) [reply client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss]: Right but maybe with `break-small` mixin? :-) 

https://github.com/WordPress/gutenberg/blob/e16260ff98ee4c3163332d7659b50c4fc7a78320/packages/base-styles/_mixins.scss#L35

Not a biggie tho and can be merged without.

**Pull request merged**